tests/alembic_migrations/test_warnings.py

```diff
@@ -0,0 +1,21 @@
+import warnings
+import pytest
+
+from sqlalchemy.exc import SAWarning
+from .alembicverify_util import prepare_schema_from_migrations
+
+
+@pytest.mark.usefixtures("new_db_left")
+def test_warnings(
+    uri_left, alembic_config_left, db_setup, env_config
+):
+    with warnings.catch_warnings(record=True) as ws:
+        warnings.simplefilter("always")
+        engine, script = prepare_schema_from_migrations(
+            uri_left, alembic_config_left, db_setup=db_setup
+        )
+        print(f"{len(ws)} warnings:")
```

Comment:
Should these `print`s be removed, or are they crucial to the test?

Reply:
The prints are diagnostic for when other warnings may arise. They are not crucial to the test. I will comment them out.